OpenEdXMobile/res/layout/discussion_response_action_bar_layout.xml

```diff
@@ -2,6 +2,7 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/discussion_actions_bar"
```

Comment:
> You don't need to define an ID for the root view, as you can get it directly as the return value of the inflate() method.

Since its included in every layout, we never inflate it ourselves. So assigning an id is required.